modules/parsers/parser-hdbschema/com.sap.xsk.parser.hdbschema/src/main/antlr4/com/sap/xsk/parser/hdbschema/core/Hdbschema.g4

```diff
@@ -0,0 +1,13 @@
+grammar Hdbschema;
+
+hdbschemaDefinition: property+;
+property: schemaProp;
+
+schemaProp: 'schema' EQ STRING SEMICOLON;
+
+STRING:  '"' (ESC|.)*? '"';
+EQ : '=' ;
+SEMICOLON : ';' ;
+COMMA : ',' ;
+WS  :   [ \t\r\n\u000C]+ -> skip;   // toss out whitespace
+ESC : '\\"' | '\\\\';   // 2-char sequences \" and \\
```

Comment:
add rule for comments, they are allow in xs classic

LINE_COMMENT : '//' .*? '\r'? '\n' -> skip ; // Match "//" stuff '\n'
COMMENT : '/*' .*? '*/' -> skip ; // Match "/*" stuff "*/"